### PR TITLE
Fix Visualization for Query Pipeline notebook

### PR DESCRIPTION
**Pull request by DoganK01**
# Description

Fixed the visualization problem

Fixes # (issue)

## Version Bump?

Did I bump the version in the `pyproject.toml` file of the package I am updating? (Except for the `llama-index-core` package)

- [ ] Yes
- [x] No

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Added new unit/integration tests
- [ ] Added new notebook (that tests end-to-end)
- [ ] I stared at the code and made sure it makes sense

## Suggested Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added Google Colab support for the newly added notebooks.
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

File: docs/examples/pipeline/query_pipeline_sql.ipynb [hunk 1]
```diff
@@ -570,8 +570,35 @@
     "from pyvis.network import Network\n",
```
Comment (nerdai) [inline docs/examples/pipeline/query_pipeline_sql.ipynb:570]: <p>very nit: "agent_dag.html"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

**Review by nerdai — APPROVED** [review]
thanks @DoganK01

File: docs/examples/pipeline/query_pipeline_sql.ipynb (hunk 1 above)
Comment (DoganK01) [reply docs/examples/pipeline/query_pipeline_sql.ipynb:570]: yeah sorry forgot to change that name

**Comment by DoganK01** [comment]
hey @nerdai can you check the linting test ? It is about metadata I think but really dont know about this issue for colab notebooks

**Pull request merged**